### ISSUE TITLE
Can't remove Windows package (application)

### ISSUE THREAD

**Issue by alrf** [chef/chef#5580]
## Description

Briefly describe the issue
Can't remove Windows package (application)

## Chef Version

Tell us which version of chef-client you are using (see below for Server+ChefDK bugs).
Chef-client v12.7.2

## Platform Version

Tell us which Operating System distribution and version chef-client is running on.
Microsoft Windows Server 2012 R2 Standard

## Replication Case

Tell us what steps to take to replicate your problem.  See [How to create a Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve)
for information on how to create a good replication case.

In my chef recipe:
```
package 'Microsoft Visual C++ 2012 Redistributable (x86) - 11.0.60610' do
 	action :remove
end
```

## Client Output

The relevant output of the chef-client run or a link to a gist of the entire run, if there is one.

```
Mixlib::ShellOut::CommandTimeout: windows_package[Microsoft Visual C++ 2012 Redistributable (x86) - 11.0.60610] had an error: Mixlib::ShellOut::CommandTimeout: command timed out:
       ---- Begin output of start "" /wait /d"C:\ProgramData\Package Cache\{95716cce-fc71-413f-8ad5-56c2892d4b3a}\vcredist_x86.exe  " uninstall  & exit %%%%ERRORLEVEL%%%% ----
       STDOUT:
       STDERR:
       ---- End output of start "" /wait /d"C:\ProgramData\Package Cache\{95716cce-fc71-413f-8ad5-56c2892d4b3a}\vcredist_x86.exe  " uninstall  & exit %%%%ERRORLEVEL%%%% ----

       ProcessId: 3248
       app_name: C:\Windows\system32\cmd.exe
       command_line: cmd /c "start "" /wait /d"C:\ProgramData\Package Cache\{95716cce-fc71-413f-8ad5-56c2892d4b3a}\vcredist_x86.exe  " uninstall  & exit %%%%ERRORLEVEL%%%%"
       timeout: 600
```

The debug output (chef-client -l debug) may be useful, but please link to a gist, or truncate it.

## Stacktrace

Please include the stacktrace.out output or link to a gist of it, if there is one.

### NOTE: CHEF CLIENT BUGS ONLY

This issue tracker is for the code contained within this repo -- `chef-client`, base `knife` functionality (not
plugins), `chef-apply`, `chef-solo`, `chef-client -z`, etc.

* [Server issues](https://github.com/chef/chef-server/issues/new)
* [ChefDK issues](https://github.com/chef/chef-dk/issues/new)
* Cookbook Issues (see the https://github.com/chef-cookbooks repos or search [Supermarket](https://supermarket.chef.io) or GitHub/Google)

**Comment by alrf** [comment]
'/quiet' option should be added, i.e. something like that:
command_line: cmd /c "start "" /wait /d"C:\ProgramData\Package Cache\{95716cce-fc71-413f-8ad5-56c2892d4b3a}\vcredist_x86.exe  " uninstall **quiet**   & exit %%%%ERRORLEVEL%%%%"

**Comment by alrf** [comment]
Some applications has a **QuietUninstallString** key in registry. Value of this key contains command for quiet uninstall application and may be used too.

**Comment by btm** [comment]
Does this particular program create a `QuietUninstallString` registry key? Do you know what kind of installer vcredist_x86.exe is?

**Comment by alrf** [comment]
I don't remember it, but I can say that we check both keys - UninstallString and QuietUninstallString - and if we have one of them, we use it for delete application (for UninstallString we use /quiet parameter)

**Comment by btm** [comment]
This should be fixed by #5849 and in Chef 13.0.13+ or so.
